calcom/cal.com
# fix: wrong reschedule link for an org member for a booking on event outside the org

## What does this PR do?



- Fixes #14832 
- Fixes CAL-3599 (Linear issue number - should be visible at the bottom of the GitHub issue description)

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

- Login as an acme org member
- Book pro/30min
- Go to /bookings page and find that booking
- Try to reschedule
- Correct reschedule link is shown
